tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_skip_fixture.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\MethodCall\ConvertRenderTemplateShortNotationToBundleSyntaxRector\Fixture;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class BaseController extends Controller
```

Comment:
Name `skip_already_bundle_name.php.inc` and the controller `SkipAlreadyBundleName`

Reply:
Renamed the three tests, hopefully it's clear enough.